src/frontend/src/flows/recovery/confirmSeedPhrase.ts

```diff
@@ -156,6 +169,7 @@ export const wordTemplate = ({
       type="text"
       class="c-recoveryInput"
       ${ref(word.elem)}
+      ?autofocus=${word.autofocus === true}
```

Comment:
nice, I didn't know about `autofocus`! We could use it in other places too then (different PRs). Once thing though, it looks like it doesn't work on iOS: https://caniuse.com/?search=autofocus

Reply:
It is, but It I think its by design in the case of IOS, it would be a bit disturbing if the keyboard would show up all of the sudden. But if you think its still a good idea to do it on IOS as well, I can try doing in the JS way

Reply:
I get the virtual keyboard argument and don't care too strongly. If you do want to do it the js way, you can reuse this: https://github.com/dfinity/internet-identity/blob/d8a9371ec23d1771b6ec0ae8c3209799e3ac2fb5/src/frontend/src/components/anchorPicker.ts#L49

Up to you!

Reply:
@nmattia I don't min changing it to the JS way. After @peterpeterparker question I thought its kinda nice that it does not do it on IOS. About the `words.find()` I wanted to do something like that first but then thought it was less performant, but I should probably only care about performance if it becomes a problem. I am changing it to find. Its gonna be easier to understand and maintain.